presto-product-tests/src/main/resources/sql-tests/testcases/window_functions/windowDescriptionAABwithEP.sql

```diff
@@ -0,0 +1,8 @@
+-- database: presto; groups: window;
+select
```

Comment:
Here and elsewhere, the projection node is an artifact of the current implementation of the window functionality. We shouldn't name the test after an implementation detail.